src/poll.rs

```diff
@@ -537,22 +543,23 @@ struct AtomicState {
 
 const MASK_2: usize = 4 - 1;
 const MASK_4: usize = 16 - 1;
+const MASK_8: usize = 256 - 1;
 const QUEUED_MASK: usize = 1 << QUEUED_SHIFT;
 const DROPPED_MASK: usize = 1 << DROPPED_SHIFT;
 
 const READINESS_SHIFT: usize = 0;
-const INTEREST_SHIFT: usize = 4;
-const POLL_OPT_SHIFT: usize = 8;
-const TOKEN_RD_SHIFT: usize = 12;
-const TOKEN_WR_SHIFT: usize = 14;
-const QUEUED_SHIFT: usize = 16;
-const DROPPED_SHIFT: usize = 17;
+const INTEREST_SHIFT: usize = 8;
+const POLL_OPT_SHIFT: usize = 16;
+const TOKEN_RD_SHIFT: usize = 20;
+const TOKEN_WR_SHIFT: usize = 22;
+const QUEUED_SHIFT: usize = 24;
+const DROPPED_SHIFT: usize = 25;
```

Comment:
I'd be very wary of changing this here. Could you elaborate more about why you are changing these constants? I'm assuming it is to enable the AIO readiness to be used in the custom readiness queue, however, I don't think that is necessary.

I was reserving the extra bits in the atomic to potentially implement other features. I would prefer it if more readiness states didn't need to be added here.

I would guess that a better option would be to panic if invalid readiness bits are passed to a `Registration` or `SetReadiness`.